kettle-columnstore-bulk-exporter-plugin/src/main/java/com/mariadb/columnstore/api/kettle/KettleColumnStoreBulkExporterStepMeta.java

```diff
@@ -87,9 +88,20 @@
   static {
       String nativeLib = "";
   try {
-      String jar = KettleColumnStoreBulkExporterStepMeta.class.getProtectionDomain().getCodeSource().getLocation().toURI().getPath();
-      String jarPath = jar.substring(0, jar.lastIndexOf(File.separator));
-      nativeLib = jarPath + File.separator + "lib" + File.separator + "libjavamcsapi.so";
+      String jar = new URI(KettleColumnStoreBulkExporterStepMeta.class.getProtectionDomain().getCodeSource().getLocation().toString().replace(" ", "%20")).getPath();
+      System.err.println(jar);
+      String jarPath = jar.substring(0, jar.lastIndexOf("/"));
+      if(System.getProperty("os.name").indexOf("nux") >= 0){
+          nativeLib = jarPath + File.separator + "lib" + File.separator + "libjavamcsapi.so";
+      }
+      //On Windows try to load javamcsapi.dll's dependent libraries libiconv.dll, libxml2.dll, libuv.dll and mcsapi.dll from the lib dir
+      else if(System.getProperty("os.name").startsWith("Windows")){
+          nativeLib = jarPath + File.separator + "lib" + File.separator + "javamcsapi.dll";
+          try{ System.load(jarPath + File.separator + "lib" + File.separator + "libiconv.dll"); } catch(UnsatisfiedLinkError e){ System.err.println("Wasn't able to load libiconv.dll"); }
```

Comment:
could we precaclulate jarPath + File.separator + "lib" + File.separator  once and then use it 5 times. Will make the code more readable?